ui/pages/multi-srp/import-srp/import-srp.tsx

```diff
@@ -90,13 +90,7 @@ export const ImportSrp = () => {
 
   async function importWallet() {
     if (isSocialLoginEnabled) {
-      const isPasswordOutdated = await dispatch(
-        actions.checkIsSeedlessPasswordOutdated(),
-      );
-      if (isPasswordOutdated) {
-        await actions.forceUpdateMetamaskState(dispatch);
-        return;
-      }
+      await dispatch(actions.checkIsSeedlessPasswordOutdated());
```

Comment:
do u need to force return here? or does force state update reload the entire app?